File: regress/nodejs/update-badref.ts
const db: ortdb = ort(dbfile);
const ctx: ortctx = db.connect();

const id1: bigint = ctx.db_bar_insert();
if (id1 < 0)
	return false;

const id2: bigint = ctx.db_foo_insert(id1);
if (id2 < 0)
	return false;

if (ctx.db_foo_update_barid(1242))
	return false;

return true;

File: regress/rust/update-badref.rs
use orb::ort;
use std::env;

fn main() {
    let args: Vec<String> = env::args().collect();
    assert_eq!(args.len(), 2);
    let db = ort::Ortdb::new(&args[1]);
    let ctx = db.connect().unwrap();

    let id1 = ctx.db_bar_insert().unwrap();
    assert_ne!(id1, -1);

    let id2 = ctx.db_foo_insert(id1).unwrap();
    assert_ne!(id2, -1);

    assert_ne!(ctx.db_foo_update_barid(1234).unwrap(), true);
}
